Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -1077,99 +1071,216 @@ private protected void UpdateMaterialAssetByDefault(MaterialData src, ModelImpor
+
+                    HandleKeywords(destMat, textureHolders, prop, MeshSyncConstants._EMISSION);
+                }
+            }
+            else if (propName == MeshSyncConstants._MetallicGlossMap) {
+                HandleKeywords(destMat, textureHolders, prop, MeshSyncConstants._METALLICGLOSSMAP);
+                HandleKeywords(destMat, textureHolders, prop, MeshSyncConstants._METALLICSPECGLOSSMAP);
+            }
+            else if (propName == MeshSyncConstants._BumpMap) {
+                HandleKeywords(destMat, textureHolders, prop, MeshSyncConstants._NORMALMAP);
+            }
+            else if (propName == MeshSyncConstants._ParallaxMap) {
+                HandleKeywords(destMat, textureHolders, prop, MeshSyncConstants._PARALLAXMAP);
+            }
+#if AT_USE_HDRP
+            else if (propName == MeshSyncConstants._EmissiveColorMap) {
+                Color baseEmissionColor = Color.white;
+                var emissionColorProp = FindMaterialPropertyData(materialProperties, MeshSyncConstants._EmissionColor);
+                if (emissionColorProp.HasValue) {
+                    baseEmissionColor = emissionColorProp.Value.vectorValue;
+                }
+
+                var emissionStrengthProp = FindMaterialPropertyData(materialProperties, MeshSyncConstants._EmissionStrength);
+                float emissionStrength = 0;
+                if (emissionStrengthProp.HasValue) {
+                    emissionStrength = emissionStrengthProp.Value.floatValue;
+                }
+                 
+                destMat.SetFloat(MeshSyncConstants._UseEmissiveIntensity, 1);
+                destMat.SetFloat(MeshSyncConstants._EmissiveIntensityUnit, 0);
+                destMat.SetFloat(MeshSyncConstants._EmissiveIntensity, emissionStrength);
+                destMat.SetColor(MeshSyncConstants._EmissiveColorLDR, baseEmissionColor);
+                destMat.SetColor(MeshSyncConstants._EmissiveColor, baseEmissionColor * emissionStrength);
+            }
+            else if (propName == MeshSyncConstants._HeightMap) {
+                if (prop.type == MaterialPropertyData.Type.Texture) {
+                    MaterialPropertyData.TextureRecord rec = prop.textureValue;
+                    Texture2D                          tex = FindTexture(rec.id, textureHolders);
+                    if (tex != null) {
+                        destMat.EnableKeyword(MeshSyncConstants._HEIGHTMAP);
+
+                        // If there is no displacement mode set, set it to vertex displacement:
+                        if (destMat.IsKeywordEnabled( "_PIXEL_DISPLACEMENT")) {
+                            destMat.EnableKeyword("_VERTEX_DISPLACEMENT");
+                            destMat.SetInt("_DisplacementMode", 1);
                         }
-                        break;
-                    default: break;
+
+                        float scale = 10;
```

Comment:
Why is 10 the default value ?

Reply:
It's a small fallback value in case the node setup in blender does not specify the height scale. Normally this is not used because the value comes from the displacement node.